lib/webpack-config-bower.js

```diff
@@ -30,7 +30,7 @@ module.exports = ({
 			// the order is respected
 			modules: [path.join(installationDirectory, "bower_components")],
 			// These JSON files are read in directories
-			descriptionFiles: ["bower.json"],
+			descriptionFiles: [".bower.json"],
```

Comment:
It would be good to have a comment here explaining what the dot is, where it comes from, and why it's preferable to using `bower.json`

Reply:
https://github.com/Financial-Times/origami-build-service-v3/pull/35/commits/ae5ac1a130a08d0bfb99f7b36b092459177bff4f

Reply:
<3!